web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -427,63 +489,123 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
-                            onClick={() => confirmDelete(id)}
-                            edge="end"
-                            aria-label="Delete"
-                          >
-                            <Delete />
-                          </IconButton>
-                        </Tooltip>
-                      </>
-                    )
-                  }
-                  key={`period-${id}`}
-                >
-                  <ListItemAvatar
-                    key={`period-lia-${id}`}
-                    onClick={() => onPeriodClick(id)}
+                            <IconButton
+                              onClick={() => toggleReviewPeriod(id)}
+                              aria-label={
+                                reviewStatus === ReviewStatus.OPEN
+                                  ? 'Archive'
+                                  : 'Unarchive'
+                              }
+                            >
+                              {reviewStatus === ReviewStatus.OPEN ? (
+                                <Archive />
+                              ) : (
+                                <Unarchive />
+                              )}
+                            </IconButton>
+                          </Tooltip>
+                          <Tooltip title="Delete">
+                            <IconButton
+                              onClick={() => confirmDelete(id)}
+                              edge="end"
+                              aria-label="Delete"
+                            >
+                              <Delete />
+                            </IconButton>
+                          </Tooltip>
+                        </>
+                      )
+                    }
+                    key={`period-${id}`}
                   >
-                    <Avatar>{reviewStatusIconMap[reviewStatus]}</Avatar>
-                  </ListItemAvatar>
-                  <ListItemText
-                    key={`period-lit-${id}`}
-                    onClick={() => onPeriodClick(id)}
-                    primary={`${name} - ${titleCase(reviewStatus)}`}
-                    secondary={getSecondaryLabel(id)}
-                  />
-                </ListItem>
-              </>
-            ))
+                    <ListItemAvatar
+                      key={`period-lia-${id}`}
+                      onClick={() => onPeriodClick(id)}
+                    >
+                      <Avatar>{reviewStatusIconMap[reviewStatus]}</Avatar>
+                    </ListItemAvatar>
+                    <ListItemText
+                      key={`period-lit-${id}`}
+                      onClick={() => onPeriodClick(id)}
+                      primary={`${name} - ${titleCase(reviewStatus)}`}
+                      secondary={getSecondaryLabel(id)}
+                    />
+                    <div className="datePickerFlexWrapper">
+                      <DatePickerField
+                        date={dayjs(launchDate)}
```

Comment:
What happens if you change this line to `date={launchDate}` ?
Maybe you don't need the `dayjs` function.
Same on lines 575 and 590.

Reply:
This was ok to remove as long as we use null for empty values.